messages.lua

```diff
@@ -0,0 +1,185 @@
+local Messages = {}
+BuffDuty.Messages = Messages
+
+local setToDefault = "default"
+
+local defaultPublicTitle = "Dear $class$s, please support our raid with your buffs, love and care!"
+local defaultDutyLine = "Group$s $groups - {rt$i} $name {rt$i}"
+local defaultDutyWhisper = "Thank you $name, for supporting our raid today! Would you kindly attend to buffing group$s $groups when you are able."
+local defaultSingleTitle = "Looks like we only have one $class in the raid today. {rt1}$name{rt1}, dear, would you kindly provide everyone with your wonderful buffs!"
+local defaultSingleWhisper = "Dear $name, looks like you are the only $class in the raid today, would you kindly provide everyone with your wonderful buffs!"
+
+function Messages:Initialise()
+    -- Fixed messages
+    self.message_title = "(Buff Duty) • %s •"
+    -- Settable messages
+    self.public_title = defaultPublicTitle
+    self.duty_line = defaultDutyLine
+    self.duty_whisper = defaultDutyWhisper
+    self.single_title = defaultSingleTitle
+    self.single_whisper = defaultSingleWhisper
+end
+
+-- Validation functions return true = OK / false = Error and an error string
+local function validatePublicTitle(public_title)
+    if public_title == setToDefault then
+        return true
+    end
+    return true
```

Comment:
return false?

Reply:
Nope, `return true` is correct. 
Public Title and Single Whisper are just generic strings with no specific need for $name, $groups, etc. The validation functions are there to match the pattern of the other `validate` functions, which enables code such as Console to call them without worrying about the specifics or making special exceptions.

Reply:
But i Still don't understand... This function always return true, right?